src/jit/instr.h

```diff
@@ -277,23 +277,19 @@ enum emitAttr : unsigned
 enum InstructionSet
 {
 #ifdef _TARGET_XARCH_
-    // Linear order start
     InstructionSet_ILLEGAL = 0,
+    // Start linear order SIMD instruction sets
+    // These ISAs have strictly gen-to-gen order.
```

Comment:
Can you please spell out "gen-to-gen"? I think that would be "generation to generation"? Maybe it would be more clear, "Each ISA is a superset of the previous ISA"? (Is that true?)

Reply:
> I think that would be "generation to generation"?  

Yes, will do.

> "Each ISA is a superset of the previous ISA"? (Is that true?)

Good point, I think we could read "Each SIMD level is a superset of the previous levels".

Reply:
>  "Each ISA is a superset of the previous ISA"? (Is that true?)

Not really, but we can say this for `SIMDLevel`.